### ISSUE TITLE
A small question on unit velocity field

### ISSUE THREAD

**Issue by xiaorugao999** [cwmok/LapIRN#33]
Hello, thank you for your perfect work. I have a small question I'd like to ask you. Why do we multiply by (z-1) instead of (z-1)/2 when smoothing the velocity field?
<img width="469" alt="image" src="https://github.com/cwmok/LapIRN/assets/69455175/76f61867-acdf-4321-852b-8d7a3ad46a02">

**Comment by cwmok** [comment]
@xiaorugao999 
You're correct. The correct way should be * (z-1)/2. But this work was my very preliminary work and all the results in the paper are reported using this code. I tend to preserve this minor mistake for better reproducibility.

Thanks for pointing it out.

**Comment by xiaorugao999** [comment]
@cwmok 
Thank you for your timely response. I have recently encountered some issues with deformation field normalization, and I have learned a lot through your code. Once again, thank you for your excellent work.
